yt_dlp/extractor/dplay.py

```diff
@@ -65,6 +65,7 @@ def _download_video_playback_info(self, disco_base, video_id, headers):
         return streaming_list
 
     def _get_disco_api_info(self, url, display_id, disco_host, realm, country, domain=''):
+        country = self.get_param('geo_bypass_country') or country
```

Comment:
Since this a worldwide site, it doesn't make sense to hardcode a `country`, as is required in the `_DISCO_API_PARAMS` var for extractors subclassed from `DiscoveryPlusBaseIE`:
https://github.com/yt-dlp/yt-dlp/blob/9c13de0dd2e24211ffd5eac7da1388107d2cc7be/yt_dlp/extractor/dplay.py#L1029-L1033

With the change to `_get_disco_api_info`, the user can use `--geo-bypass-country` to set their country param (and `DPlayBaseIE` also uses this country value for xff anyways).

Letting the user configure the `country` param would also partially address problems reported in #2138 and #2361 (though completely resolving those issues is outside the scope of this PR), see:
https://github.com/yt-dlp/yt-dlp/issues/2361#issuecomment-1549966571
https://github.com/yt-dlp/yt-dlp/issues/2138#issuecomment-1546623404

Reply:
But the `disco_host` may also vary by country (perhaps not in this case)?

Reply:
it did not vary for this site; that's something we can deal with when the extractor at hand demands it